src/Interfaces/IArtist.ts

```diff
@@ -0,0 +1,15 @@
+export interface IArtist {
```

Comment:
May I ask why you've used an interface here instead of a class? This is something I've seen in other Angular projects but it's strange to me because this interface has no public methods, all it has is properties.